Week-1/Session-02-HTML_Deployment/Lab/text_basics.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <!-- Text tags -->
+    <h1>H1 Tag:</h1>
+    <h2>H2 Tag</h2>
+    <p>This is a paragraph.</p>
+    <!-- Text formats -->
+    <em>This text is emphasized.</em>
+    <strong>This text is strong.</strong>
+    <u>This text is underlined.</u>
+
+    <!-- Div and span with text -->
+    <h1>Div and span with text:</h1>
+    <p>This text inside
+        <div>
+            <p>a div tag. Hence div is a block container</p>
```

Comment:
Its okay, but going forward if you have a single line in the div tag, dont use any another tag inside div. 


As here, both div and p are block scope level tag, so adding P tag will not add any advantage here.